docs/developer-guides/merkle-root-construction.md

```diff
@@ -2,10 +2,45 @@
 
 ---
 
-#### Merkle Trees
+## Merkle Trees
 
-The merkle root is constructed using all the TXIDs of transactions in
-this block, but first the TXIDs are placed in order as required by the
+
+A merkle tree is a hash-based data structure that summarizes all the transactions in a block. It can be used to quickly and efficiently verify whether or not a transaction is included in a block. 
+
+Merkle trees are created by repeatedly hashing pairs of nodes until there is only one hash left, the Merkle Root. They are constructed from the bottom up, from hashes of the leaf nodes (hashes of the raw input data). The diagram below shows a merkle root constructed from eight leaf nodes. 
+
+![Merkle Tree](../img/core-blockchain-concepts/merkle_tree.svg)
+
+When calculating the merkle roots included in the block header, the data items are the transaction IDs of the transactions in the block.
+
+Regular transactions and stake-based transactions (ticket purchases, votes, revocations) have separate merkle trees. Regular transactions are hashed to the `MerkleRoot` tree. Stake-based transactions are hashed to the `StakeRoot` tree. 
```

Comment:
We should be aware that this will need to be updated -  this changes when DCP-0005 activates: https://github.com/davecgh/dcps/blob/dcp0005/dcp-0005/dcp-0005.mediawiki#Abstract

Reply:
Created Issue to track: https://github.com/jholdstock/dcrdevdocs/issues/39